scheduler/src/cook/mesos/mesos_mock.clj

```diff
@@ -275,8 +295,12 @@
           requested-resources (combine-resources (mapcat :resources tasks))
           resources' (subtract-resources available-resources requested-resources)
           tasks (map #(assoc % :launched-time (t/now)) tasks)
-          task->runtime-ms (-> state :config :task->runtime-ms)]
-      (log/info "Resources requested by tasks: " {:requested-resources requested-resources})
+          task->runtime-ms (-> state :config :task->runtime-ms)
+          new-time-task-id-pairs (map (fn [{:keys [task-id] :as task}]
```

Comment:
That would certainly make the filter on kill easier =)

I'm reasonably indifferent though inertia is leading me to lean toward keeping it as is. What is your reasoning why?